vespaclient-container-plugin/src/main/resources/configdefinitions/document-operation-executor.def

```diff
@@ -1,15 +1,15 @@
 # Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 package=com.yahoo.document.restapi
 
-# Delay before a throttled operation is retried.
-resendDelayMillis     int default=100
-
-# Time between a document operation is received and a timeout response is sent
-defaultTimeoutSeconds int default=180
-
-# Time after which a visitor session times out
-visitTimeoutSeconds   int default=120
+# Duration for which resender thread sleeps after an operation is throttled
+resendDelayMillis     int default=10
 
 # Bound on number of document operations to keep in retry queue — further operations are rejected
 maxThrottled          int default=1000
 
+# Whether to perform dispatch from Jetty threads
+doDispatchWithEnqueue bool default=true
+
+# Number of workers in the resender thread pool
+resenderCount         int default=1
```

Comment:
We will probably want to control resender threads from `ContainerThreadpool` later on.

Reply:
Yes, this isn't even used now >_< Let's clean up when we decide on how we do it.